Tests/StreamChatUITests/UISDKdocumentation/CGPoint+Move.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+extension CGPoint {
+    mutating func move(x: CGFloat? = nil, y: CGFloat? = nil) {
+        if let x = x {
+            self.x += x
+        }
+
+        if let y = y {
+            self.y += y
+        }
+    }
```

Comment:
nit: we have a `CGPoint.offsetBy(x:, y:)` which does the same

Reply:
Thanks, I deleted the extension, let's not bring unnecessary code 😅